From 22eb919f1b7900e2af0e952b1f1bee522f612b4a Mon Sep 17 00:00:00 2001
From: Mikulas Patocka <mpatocka@redhat.com>
Date: Thu, 16 Jul 2026 16:02:48 +0200
Subject: [PATCH 05138/15333] dm-inlinecrypt: don't overwrite the error with
 -EINVAL

get_key_size already returns -EINVAL on error, so we don't have to
overwrite it again. No functional change.

Signed-off-by: Mikulas Patocka <mpatocka@redhat.com>
---
 drivers/md/dm-inlinecrypt.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/md/dm-inlinecrypt.c b/drivers/md/dm-inlinecrypt.c
index 41293c18d10f..863cb471e95e 100644
--- a/drivers/md/dm-inlinecrypt.c
+++ b/drivers/md/dm-inlinecrypt.c
@@ -347,7 +347,6 @@ static int inlinecrypt_ctr(struct dm_target *ti, unsigned int argc, char **argv)
 	err = get_key_size(&argv[1]);
 	if (err < 0) {
 		ti->error = "Cannot parse key size";
-		err = -EINVAL;
 		goto bad;
 	}
 	ctx->key_size = err;
-- 
2.55.0

